xmodule/video_block/transcripts_utils.py

```diff
@@ -1041,6 +1044,8 @@ def get_transcript_from_learning_core(video_block, language, output_format, tran
     """
     Get video transcript from Learning Core.
 
+    Limitation: This is only going to grab from the Draft version.
+
     HISTORIC INFORMATION FROM WHEN THIS FUNCTION WAS `get_transcript_from_blockstore`:
```

Comment:
Would it be fair to delete this Historic Information or update it for Learning Core?

Reply:
Oh right, I should update it.